packages/asc-ui/src/components/FilterTag/FilterTagStyle.ts

```diff
@@ -0,0 +1,36 @@
+import styled, { css, styledComponents } from '@datapunt/asc-core'
+import { Props as TypographyProps } from '../Typography/TypographyStyle'
+import Typography from '../Typography'
+import { themeColor } from '../../utils'
+import IconStyle from '../Icon/IconStyle'
+
+export type Props = {} & TypographyProps & styledComponents.StyledProps<any>
+
+export const FilterTagStyleCSS = () => {
+  return css<Props>`
+    margin: 0 8px 8px 0;
```

Comment:
Have a look at the utility `themeSpacing` and please check with a designer if you need to use a different spacing than one that fits the 4 pixel grid